build.gradle.kts

```diff
@@ -0,0 +1,161 @@
+        relocate("com.iwebpp.crypto", "github.scarsz.discordsrv.dependencies.iwebpp.crypto")
+        relocate("com.vdurmont.emoji", "github.scarsz.discordsrv.dependencies.emoji")
+        relocate("com.neovisionaries.ws", "github.scarsz.discordsrv.dependencies.ws")
+        relocate("net.kyori", "github.scarsz.discordsrv.dependencies.kyori")
+        relocate("dev.vankka.mcdiscordreserializer", "github.scarsz.discordsrv.dependencies.mcdiscordreserializer")
+        relocate("dev.vankka.simpleast", "github.scarsz.discordsrv.dependencies.simpleast")
+        relocate("org.bstats", "github.scarsz.discordsrv.dependencies.bstats")
+        relocate("org.apache.commons", "github.scarsz.discordsrv.dependencies.commons")
+        relocate("org.json.simple", "github.scarsz.discordsrv.dependencies.json.simple")
+        relocate("org.json", "github.scarsz.discordsrv.dependencies.json")
+        relocate("org.minidns", "github.scarsz.discordsrv.dependencies.minidns")
+        relocate("okhttp3", "github.scarsz.discordsrv.dependencies.jda")
+        relocate("okio", "github.scarsz.discordsrv.dependencies.okio")
+        relocate("gnu.trove", "github.scarsz.discordsrv.dependencies.trove")
+        relocate("com.fasterxml.jackson", "github.scarsz.discordsrv.dependencies.jackson")
+        relocate("com.google.common", "github.scarsz.discordsrv.dependencies.google.common")
+        relocate("com.google.errorprone", "github.scarsz.discordsrv.dependencies.google.errorprone")
+        relocate("com.google.gson", "github.scarsz.discordsrv.dependencies.google.gson")
+        relocate("com.google.j2objc", "github.scarsz.discordsrv.dependencies.google.j2objc")
+        relocate("com.github.kevinsawicki", "github.scarsz.discordsrv.dependencies.kevinsawicki")
+        relocate("com.github.zafarkhaja", "github.scarsz.discordsrv.dependencies.zafarkhaja")
+        relocate("alexh", "github.scarsz.discordsrv.dependencies.alexh")
+
+        // Merge META-INF/services files where needed
+        mergeServiceFiles()
+
+        // Exclude signatures, maven/ and proguard/ from META-INF
+        exclude("META-INF/*.SF")
+        exclude("META-INF/*.DSA")
+        exclude("META-INF/*.RSA")
+        exclude("META-INF/maven/**")
+        exclude("META-INF/proguard/**")
+    }
+}
+
+fun configureKotlin(kotlinCompile: Action<KotlinCompile>) {
+    tasks.withType<KotlinCompile>().configureEach(kotlinCompile)
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    maven("https://hub.spigotmc.org/nexus/content/repositories/snapshots/")
+    maven("https://papermc.io/repo/repository/maven-public/")
+    maven("https://oss.sonatype.org/content/repositories/central")
+    maven("https://oss.sonatype.org/content/repositories/snapshots")
+    maven("https://nexus.scarsz.me/content/groups/public/") {
+        content {
+            includeGroup("github.scarsz")
+            includeGroup("me.scarsz")
+            includeGroup("me.minecraftauth")
+        }
+    }
```

Comment:
Aren't like over half of the plugin integrations coming from this repo? They should be included in the `content {}`-block or it should be removed

Reply:
Copied that over from Ascension's build script but you're right